### ISSUE TITLE
R cmd check fails when trying to build vignette

### ISSUE THREAD

**Issue by qdread** [mweinroth/epi2me2r#8]
Hi Maggie I just tried to run the `R CMD check` on my own system (Windows 10, R 4.1.2) but it fails because of an error in the vignette build.

```
E  creating vignettes (26s)
   --- re-building 'epi2me2r-vignette.Rmd' using rmarkdown
   Quitting from lines 96-100 (epi2me2r-vignette.Rmd) 
   Error: processing vignette 'epi2me2r-vignette.Rmd' failed with diagnostics:
   metadata does not have columns named "arma_filename" and "arma_barcode".
```

That was the block of code that runs `amr_raw_to_phyloseq`. Looks like there is still maybe an inconsistent spelling of arma and amra?

**Comment by qdread** [comment]
I also tried `devtools::test()` and it fails for the same reason:

```
Error (test-amr_raw_to_phyloseq.R:4:3): amr_raw_to_phyloseq gives phyloseq object
Error in `amr_raw_to_phyloseq(path.to.amr.files = "testdata/example_amr_data/", 
    metadata = metadata)`: metadata does not have columns named "arma_filename" and "arma_barcode".
```

**Comment by qdread** [comment]
I think I solved this problem! The file `example_metadata.csv` had a non standard file encoding so I went into excel and did Save As to convert it from a UTF-8 CSV to a regular CSV. So this issue should be resolved by the commit 0ccd9a7 that I just made. 

Also notice in that same commit that I went through the vignette and corrected the way that the example files are loaded. They should not be loaded from `inst/extdata` directly because that will not be on the user's file system. If someone else installs the package, you have to use the `system.file()` function to find where the installed package files ended up getting installed to on their file system.

Based on all those changes I was able to eliminate all errors and warnings in the R CMD check!
